plugins/newsdownloader.koplugin/main.lua

```diff
@@ -145,8 +145,21 @@ end
 function NewsDownloader:loadConfigAndProcessFeeds()
     if not NetworkMgr:isOnline() then
         wifi_enabled_before_action = false
-        NetworkMgr:promptWifiOn()
-        return
+        NetworkMgr:turnOnWifi()
+        local timeout = 60;
+        local retry_count = 0;
+        local info = InfoMessage:new{ text = T(_("Enabling WiFi, waiting for Internet connection...\nTimeout %1 seconds."), timeout)}
```

Comment:
`Enabling Wi-Fi. Waiting for… etc.` But on platforms where we use our own https://github.com/koreader/lj-wpaclient this'd just get in the way.

Reply:
Why? what you mean by that?

Reply:
I mean that besides being a comma splice, this seems like an undesirable addition — at least in its current form — regardless of orthographic and stylistic issues.

Here's what it looks like on Kobos, for example: https://vimeo.com/244429050